packages/app/prisma-utils/src/prismaTransaction.ts

```diff
@@ -104,11 +109,13 @@ const calculateRetryDelay = (
 	return Math.min(expDelay, maxDelayMs)
 }
 
-const isRetryAllowed = <T>(result: PrismaTransactionReturnType<T>, dbDriver: DbDriver): boolean => {
+type IsRetryAllowedResult = boolean | 'increase-timeout'
```

Comment:
🟢 I think returning something like:
```
{
  retryAllowed: boolean;
  increaseTimeout: boolean;
}
```
would be a bit more intuitive. But it can be just my preference, so no strong opinion about it. 
WDYT?

Reply:
That sounds even better! I will apply your suggestion to the final version :D thanks 🙏